test/v1/connectors/DeleteForeignReliefsConnectorSpec.scala

```diff
@@ -46,7 +46,7 @@ class DeleteForeignReliefsConnectorSpec extends ConnectorSpec {
         val outcome = Right(ResponseWrapper(correlationId, ()))
 
         willDelete(
-          url = s"$baseUrl/income-tax/reliefs/foreign/${request.nino.nino}/$taxYear"
+          url = s"$baseUrl/income-tax/reliefs/foreign/${request.nino}/$taxYear"
```

Comment:
Since we're testing that the url is correct we probably shouldn't rely on toString being implemented as it is. (The test will pass if the toString override in Nino is removed.) 
Same change to other ConnectorSpecs (just the Delete ones it seems).
```suggestion
          url = s"$baseUrl/income-tax/reliefs/foreign/$nino/$taxYear"
```